### PR TITLE
WIP: Install minimal `lsb_release`

### PR DESCRIPTION
**Pull request by jakirkham**
Installs `lsb_release` to aid the `conda` packaged `gcc` post link scripts. This is prompted by discussion in gitter from [start](https://gitter.im/conda-forge/conda-forge.github.io?at=57cd87cd780310286bb3ac1a) to [end](https://gitter.im/conda-forge/conda-forge.github.io?at=57cd88d5cdbf820f7f9c5e4e). The origin of this call to `lsb_release` in the `gcc` package comes from PR ( https://github.com/conda/conda-recipes/pull/570 ).

Basically the same as PR ( https://github.com/conda-forge/docker-images/pull/34 ). However, this is the minimal version. It only installs `redhat-lsb-core` and drops all the dependencies intentionally. The `lsb_release` command seems to work fine from testing. So this should still solve the intended problem.

cc @astrofrog @ocefpaf @msarahan @pelson @jjhelmus

### CODE REVIEW

**Comment by jakirkham** [comment]
Just went through this and everything seems to work fine all `lsb_release` options work.

As we are excluding the dependencies when downloading this, went through and inspected the RPM we are [downloading]( http://centos.chicago.waneq.com/6.8/os/x86_64/Packages/redhat-lsb-core-4.0-7.el6.centos.x86_64.rpm ). Everything is either statically linked or is a symlink to something already in the system. The RPM basically just includes `lsb_release` and a couple other very basic binaries.

Therefore this seems to be the best option. Going to close out PR ( https://github.com/conda-forge/docker-images/pull/34 ).

Please let me know your thoughts on this.

**Comment by jakirkham** [comment]
This works, but it is a bit noisy with the warnings. Not sure how to convince it to stop ATM.

**Comment by mbargull** [comment]
Closing as per https://github.com/conda-forge/docker-images/pull/38#issuecomment-507262747.

**Comment by maresb** [comment]
`lsb_release` is now available via https://github.com/conda-forge/lsb-release-feedstock/
